.registry/behavior.yaml

```diff
@@ -1,8 +1,8 @@
 source:
   path: "kustomize"
 crd:
-  kind: MinimalGCP
-  apiVersion: gcp.resourcepacks.crossplane.io/v1alpha1
+  kind: SampleGCP
+  apiVersion: gcp.stacks.crossplane.io/v1alpha1
```

Comment:
What are your thoughts on putting `samples` in the group so that all of the sample stacks roll into the same group at that level? Example: `gcp.samples.stacks.crossplane.io`

Reply:
i like this idea

Reply:
That seems fine to me, I could also see wanting it to roll-up to `gcp` before `samples` though

Reply:
@suskin @rathpc what you you call the `Kind` in this case? It seems like `gcpsample.gcp.samples.stack.crossplane.io` is pretty repititive

Reply:
i think due to that, `gcpsamples.stacks.crossplane.io` is probably fine

Reply:
i think we can just leave this as is for now